install/upgrade.php

```diff
@@ -205,5 +205,48 @@ protected function upgrade_2_7() {
 	 */
 	protected function upgrade_3_4() {
 		delete_transient( 'pll_languages_list' );
+
+		// Migrate locale fallbacks from term metas to language term description.
+		$terms = get_terms(
+			array(
+				'taxonomy'   => 'language',
+				'hide_empty' => false,
+				'meta_query' => array( // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query
+					array(
+						'key'         => 'fallback',
+						'compare_key' => 'EXISTS',
+					),
```

Comment:
I thought to this too. I did not include it in my proposal because it doesn't simplify the code afterwards. But why not.